.github/dependabot.yml

```diff
@@ -5,6 +5,10 @@
 
 version: 2
 updates:
+  - package-ecosystem: "gitsubmodule" # See documentation for possible values
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
The ticket specifies "weekly", not "daily".  I don't feel terribly strongly, but less frequent updates might mean that any transient DET failures don't impact our CI?  Oftentimes, any failures caused by DET changes are fixed within a few days of them being introduced.